frontend/src/routes/User/ForetagsbiljettList.js

```diff
@@ -79,27 +79,28 @@ export default class ForetagsbiljettList extends Component {
     const { biljetter } = this.props;
 
     this.props.onChange(
-      biljetter.map((value, i) => {
+      biljetter.map((number, i) => {
```

Comment:
We should rename this to `oldTicket` or something, got confused reading this code now. `number` makes very little sense here